recipes/_compile.rb

```diff
@@ -78,6 +78,9 @@
   #   https://github.com/CentOS/sig-cloud-instance-images/issues/4
   #
   package 'tar'
+  # Don't introduce a bug on RHEL by not having bzip2
+  # this was fixed previously in omnibus v2.4.12
+  package 'bzip2'
```

Comment:
@scotthain Removing the comment seems reasonable.  If we break ourselves again by removing the install, we can add it back with a comment :-)